v2.11.x/sof-ipc4-v2.11.1/arl/intel-signed/sof-arl.ri

```diff
@@ -1 +1 @@
-../../mtl/sof-mtl.ri
\ No newline at end of file
+../../mtl/intel-signed/sof-mtl.ri
```

Comment:
`../../mtl/sof-mtl.ri` is pointing to `../../mtl/intel-signed/sof-mtl.ri`, right?

Reply:
@ujfalusi ack, no functional difference, but just looking at convention, we already have communty/sof-foo.ri pointing to community/sof-bar.ri in case same binaries can be used, so to keep consistency, we should do the same for intel-signed/sof-foo.ri.